rca/utils/blocks.py

```diff
@@ -103,6 +104,46 @@ class GalleryBlock(blocks.StructBlock):
     author = blocks.CharBlock(required=False)
     link = blocks.URLBlock(required=False)
     course = blocks.CharBlock(required=False)
+    document = DocumentChooserBlock(
+        required=False, help_text="Recommended maximum file size: 10MB"
```

Comment:
**Nitpick**

'Recommended' feels a bit misleading to me given that the validation below won't accept any file above 10MB. Maybe we can just remove the word instead